### ISSUE TITLE
Can't figure out how to use the site for multiple reasons

### ISSUE THREAD

**Issue by samhatoum** [airpair/airpair-com-v1-issue-tracker#45]
Please take all the below with a pinch of salt, they are my experience and my thoughts:

I want to offer myself as an expert. It's not obvious for me to do so

I don't have any idea what the concept of a "stack" is but I'm being asked to create one. Stuff like this needs to be spelled out and the value of it to me.

I don't see the reason to anything right away. I can see there are workshops and there are posts, which sort of tells me "if you haven't got anything better to do, why not add content to our site for us". Perhaps if something like this is put as "why not market yourself".

I feel like you need "want to sell your skills? Do this" and "need some help? Do this". You are an exchange and therefore you connect a two sided market. When I come in to your site, I need to choose which side of the market I'm on, and then you ought to help me get setup as quickly as possible so I can either make money or get the help I need.

Simply put, I think you have a well designed (pretty) site and a good idea, but your UX needs work to really guide the user. Take yourselves out of the "what we want to offer" perspective and put yourselves in the "what does the user want and how do they get there"

Just my 2c :)

**Comment by jkresner** [comment]
Thanks for your extensive feedback! Please find $10 credited to your account on your billing page!

> I don't have any idea what the concept of a "stack" is but I'm being asked to create one. Stuff like this needs to be spelled out and the value of it to me.

Thanks, we'll make this clearer. Truthfully most of the value will be more visible in February as we fill in some of the features this is powering.

> Simply put, I think you have a well designed (pretty) site and a good idea, but your UX needs work to really guide the user. 

Truthfully 1/4 of the site is still to be migrated - including ALL of the expert functionality. It is scheduled for Late Jan early feb. The overall site UX for Experts will make 100x more sense in Feb and I'd like to invite you back for $50 credit to give opinions then.
